subgraph/tests/subgraph/deploy/mod.rs

```diff
@@ -0,0 +1,91 @@
+use anyhow::anyhow;
+use mustache::MapBuilder;
+use std::{fs, process::Command};
+
+pub struct Config<'a> {
+    // contracts address
+    pub contract_address: &'a String,
+    // block-number
+    pub block_number: u64,
+}
+
+pub fn deploy(config: Config) -> anyhow::Result<bool> {
+    let subgraph_template = "subgraph.template.yaml";
+    let output_path = "subgraph.yaml";
+    let root_dir = "./";
+    let end_point = "http://localhost:8020/";
+    let subgraph_name = "test/test";
+
+    let data = MapBuilder::new()
```

Comment:
why are we working with unstructured string interpolation? does serde_yaml not work?